src/point.rs

```diff
@@ -445,3 +448,26 @@ impl<S: BaseNum> fmt::Debug for Point3<S> {
         write!(f, "[{:?}, {:?}, {:?}]", self.x, self.y, self.z)
     }
 }
+
+impl<S: BaseFloat + 'static> Bound<S> for Point3<S> {
+    fn relate_plane(&self, plane: &Plane<S>) -> Relation {
+        let dist = self.dot(&plane.n);
+        if dist > plane.d {
+            Relation::In
+        }else if dist < plane.d {
+            Relation::Out
+        }else {
+            Relation::Cross
+        }
+    }
+
+    fn relate_clip_space(&self, projection: &Matrix4<S>) -> Relation {
+        use std::cmp::Ordering::*;
+        let p = projection.mul_v(&self.to_homogeneous());
+        match (p.x.abs().partial_cmp(&p.w), p.y.abs().partial_cmp(&p.w), p.z.abs().partial_cmp(&p.w)) {
```

Comment:
It would just return `Cross` for the matter of `Relation`. Not sure what would be a better way to handle it.